client/document/desktop.jsx

```diff
@@ -37,11 +36,7 @@ class Desktop extends React.Component {
 		} = this.props;
 
 		return (
-			<html
-				lang={ lang }
-				dir={ isRTL ? 'rtl' : 'ltr' }
-				className={ classNames( 'is-desktop', { 'is-fluid-width': isFluidWidth } ) }
-			>
+			<html lang={ lang } dir={ isRTL ? 'rtl' : 'ltr' } className={ classNames( 'is-desktop' ) }>
```

Comment:
Forgive me if I'm wrong, but out of curiosity, could this not just be:

`<html lang={ lang } dir={ isRTL ? 'rtl' : 'ltr' } className="is-desktop">`

Reply:
Yes, it could! But then the `wpcalypso/jsx-classname-namespace` lint rule will complain, because `is-desktop` without a base class is not valid BEM CSS.

Reply:
Indeed 👍 And also I'm sure we could use the opportunity to fix the other instances where we use `classNames()` with a single class.

Reply:
Ah, thanks for explaining! :)